tools/RAiDER/prepareCube.py

```diff
@@ -0,0 +1,254 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I'm not sure if it will be easier to merge this in and then integrate with #364 or vice versa.

Reply:
My suggestion is to merge #368 and then integrate #364. There is a common path between the cube and heightlvs option.  We only need couple of options for output projection system and spacing. The reason I wrote this separately for now was to have a prototype where dataflow is clear.